scripts/heat/ping_template.sh

```diff
@@ -47,20 +47,23 @@ chmod +x /usr/bin/ping_forever_neighbour
 cat >> /usr/bin/cinder_test << EOF
   #!/bin/sh
 
+  trap "" 1
+
   device="/dev/vdb"
   while [ ! -e \$device ]; do
       echo "waiting for \$device to be ready"
+      echo 1 > /sys/bus/pci/rescan
       sleep 10
   done
 
-  sudo mkfs -t ext4 -L cinder_volume \$device
-  sudo mount \$device /mnt
+  mkfs -t ext4 -L cinder_volume \$device
+  mount \$device /mnt
 
   while [ 1 = 1 ]; do
       date +%s >> /mnt/cinder_test.out
       sleep 1
   done
 EOF
-chmod +x /usr/bin/cinder_test
-/usr/bin/cinder_test &
+chmod +x /usr/sbin/cinder_test
+/usr/sbin/cinder_test &
```

Comment:
Strictly speaking you wouldn't need to move them as well then. They will also not work as non-root as they are writing to /var/log/ :)